create-custom-map-style/src/SamplePage.jsx

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import React, { useRef, useState, useEffect } from 'react';
+import transformRequest from './transformRequest';
+import { Auth } from 'aws-amplify';
+import { withAuthenticator } from '@aws-amplify/ui-react';
+import maplibregl from 'maplibre-gl';
+import './SamplePage.css'
+
+const SamplePage = () => {
+  const map = useRef();
+
+  const [coordinates, setCoordinates] = useState({
+    lat: 40.7356,
+    lng: -74.0541,
+    zoom: 12.44,
+  });
+
+  useEffect(async () => {
+    const credentials = await Auth.currentCredentials();
+    const { lat, lng, zoom } = coordinates;
+    const demoMap = new maplibregl.Map({
+      container: map.current,
+      center: [lng, lat],
+      zoom,
+      style: `${window.location.href || 'http://localhost:3000/'}example-style-descriptor.json`,
```

Comment:
Try `require`ing the style descriptor and setting `style` to it. That'll effectively inline the style and prevent it from needing to be served up from `public/`.

Reply:
Ok yeah I was thinking about pulling it out of the `public` dir anyway

Reply:
updated